include/z64player.h

```diff
@@ -514,7 +514,7 @@ typedef struct Player {
     /* 0x0440 */ Actor*     rideActor;
     /* 0x0444 */ u8         csMode;
     /* 0x0445 */ u8         prevCsMode;
-    /* 0x0446 */ u8         unk_446;
+    /* 0x0446 */ u8         cueId;
```

Comment:
Is this enough or would csCueId be worth it?

Reply:
not entirely sure. feel like cue is pretty unique to cutscenes, dont see actors using that term outside of them. But its also something that can be decided when cs actors are documented

Reply:
For cutscene actors or any other actor I would think cueId is enough, but the player struct is so full of things that it may be good to name things system-specifically where doable
Player.cueId is fine though, we can change it later depending on what kind of functions it's used in